frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -50,6 +53,7 @@ import {
 } from "./styled-components"
 
 import "katex/dist/katex.min.css"
+import themeColors from "../../../theme/baseTheme/themeColors"
```

Comment:
nit: I think we only use relative imports if it is on the same level, otherwise we use absolut imports.

Reply:
It's now absolut import 👍